resources/lib/play_utils.py

```diff
@@ -351,41 +351,59 @@ def send_next_episode_details(item):
-    current_item["overview"] = item_details.plot
-    current_item["tvshowtitle"] = item_details.series_name
+    current_item["art"] = {}
+    current_item["art"]["tvshow.poster"] = item_details.art.get('tvshow.poster', '')
+    current_item["art"]["thumb"] = item_details.art.get('thumb', '')
+    current_item["art"]["tvshow.fanart"] = item_details.art.get('tvshow.fanart', '')
+    current_item["art"]["tvshow.landscape"] = item_details.art.get('tvshow.landscape', '')
+    current_item["art"]["tvshow.clearart"] = item_details.art.get('tvshow.clearart', '')
+    current_item["art"]["tvshow.clearlogo"] = item_details.art.get('tvshow.clearlogo', '')
+    current_item["plot"] = item_details.plot
+    current_item["showtitle"] = item_details.series_name
     current_item["playcount"] = item_details.play_count
     current_item["season"] = item_details.season_number
     current_item["episode"] = item_details.episode_number
     current_item["rating"] = item_details.critic_rating
-    current_item["year"] = item_details.year
+    current_item["firstaired"] = item_details.year
+
 
     next_item = {}
-    next_item["id"] = next_item_details.id
+    next_item["episodeid"] = next_item_details.id
+    next_item["tvshowid"] = next_item_details.series_name
     next_item["title"] = next_item_details.name
-    next_item["image"] = next_item_details.art.get('tvshow.poster', '')
-    next_item["thumb"] = next_item_details.art.get('thumb', '')
-    next_item["fanartimage"] = next_item_details.art.get('tvshow.fanart', '')
-    next_item["overview"] = next_item_details.plot
-    next_item["tvshowtitle"] = next_item_details.series_name
+    next_item["art"] = {}
+    next_item["art"]["tvshow.poster"] = next_item_details.art.get('tvshow.poster', '')
+    next_item["art"]["thumb"] = next_item_details.art.get('thumb', '')
+    next_item["art"]["tvshow.fanart"] = next_item_details.art.get('tvshow.fanart', '')
+    next_item["art"]["tvshow.landscape"] = next_item_details.art.get('tvshow.landscape', '')
+    next_item["art"]["tvshow.clearart"] = next_item_details.art.get('tvshow.clearart', '')
+    next_item["art"]["tvshow.clearlogo"] = next_item_details.art.get('tvshow.clearlogo', '')
+    next_item["plot"] = next_item_details.plot
+    next_item["showtitle"] = next_item_details.series_name
     next_item["playcount"] = next_item_details.play_count
     next_item["season"] = next_item_details.season_number
     next_item["episode"] = next_item_details.episode_number
     next_item["rating"] = next_item_details.critic_rating
-    next_item["year"] = next_item_details.year
+    next_item["firstaired"] = next_item_details.year
 
     next_info = {
-        "current_item": current_item,
-        "next_item": next_item
+        "current_episode": current_item,
+        "next_episode": next_item
     }
 
     log.debug("send_next_episode_details: {0}", next_info)
     send_event_notification("embycon_next_episode", next_info)
 
+    play_info = {}
```

Comment:
This play info is EmbyCon specific. Would you expect each addon to supply their own play info here?

Reply:
Yes, each addon should supply whatever they want sent back to play back the data: 
https://github.com/im85288/service.upnext/wiki/Play-Info

Reply:
play_info is decided by you. Upnext accepts anything for play_info value. It will send play_info back to your addon when ready to play the next episode. For example, emby for kodi uses play_info = {'ItemIds': [123], 'PlayCommand': "PlayNow", 'ServerId': "123"}, and so up next will send that back to emby for kodi with the method "plugin.video.emby_play_action".